### ISSUE TITLE
[Feature Request] Option to group timeline by month

### ISSUE THREAD

**Issue by IndrekHaav** [pulsejet/memories#378]
Right now timeline shows photos by date. For periods when there are few photos, it's a bit much to show a separate heading for each photo, a lot of horizontal space is wasted. Maybe there could be an option to group photos by month instead? Or even disable grouping entirely and just show a continuous timeline (like how Photoprism does it)?

**Comment by pulsejet** [comment]
Grouping by months sounds like a potential option. The ideal solution to this is to show days with few photos side by side if they fit. But this is surprisingly complicated (much like the rest of the timeline implementation). Photoprism doesn't really have a "real" timeline (unless I'm missing something, it's just a virtual scroller and doesn't show dates on the scroller); implementing that would be equally challenging.

**Comment by IndrekHaav** [comment]
> Photoprism doesn't really have a "real" timeline (unless I'm missing something, it's just a virtual scroller and doesn't show dates on the scroller); implementing that would be equally challenging.

You're right, Photoprism doesn't have a timeline per se, just a continuous stream of photos, without the headings. Maybe OneDrive Photos is a better example of what I meant.
